docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -0,0 +1,155 @@
+# Flex Layout
+
+[Overview](#overview)<br>
+[FlexDirection](#flexDirection)<br>
+[FlexJustification](#flexJustification)<br>
+[AlignmentType](#alignmentType)<br>
+[FlexWrapType](#flexWrapType)<br>
```

Comment:
docs.tizen.org automatically generates TOC on the right side. 
Please remove TOC.

Reply:
Okay, Removed TOC.